### PR TITLE
[Code Cleaning]: Use "HaveLen()" instead of "len(...).To(Equal(..)" in template test

### PR DESCRIPTION
**Pull request by iholder-redhat**
**What this PR does / why we need it**:
This small PR replaces all appearences of `Expect(len(X)).To(Equal(Y))` to `Expect(X).To(HaveLen(Y))` for better readability and error presentation.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by dankenigsberg — APPROVED** [review]

**Review by jean-edouard — CHANGES_REQUESTED** [review]
Found a couple of `HaveLen(0)`, sounds like we want `BeEmpty()` instead.

File: pkg/virt-controller/services/template_test.go [hunk 1]
```diff
@@ -401,7 +401,7 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Containers[1].Image).To(Equal("some-image:v1"))
 				Expect(pod.Spec.Containers[1].ImagePullPolicy).To(Equal(kubev1.PullPolicy("IfNotPresent")))
 				Expect(*pod.Spec.TerminationGracePeriodSeconds).To(Equal(int64(60)))
-				Expect(len(pod.Spec.InitContainers)).To(Equal(0))
+				Expect(pod.Spec.InitContainers).To(HaveLen(0))
```
Comment (dankenigsberg) [inline pkg/virt-controller/services/template_test.go:404]: /lgtm cancel

Right!

**Review by dankenigsberg — CHANGES_REQUESTED** [review]

File: pkg/virt-controller/services/template_test.go (hunk 1 above)
Comment (iholder-redhat) [reply pkg/virt-controller/services/template_test.go]: Fixed.

**Comment by iholder-redhat** [comment]
> Found a couple of `HaveLen(0)`, sounds like we want `BeEmpty()` instead.

Fixed.

**Review by dankenigsberg — APPROVED** [review]

**Pull request merged**